app/src/pages/inside/dashboardItemPage/widgetsGrid/widgetsGrid.scss

```diff
@@ -0,0 +1,4 @@
+.gadget-view {
+  border: 1px solid grey;
```

Comment:
Use color variables to define colors in styles

Reply:
done